### PR TITLE
Update refreshments role for SR2023

### PR DESCRIPTION
**Pull request by Adimote**
Update refreshments for SR2023. This also includes using slightly less formal wording to make it easier to read.

### CODE REVIEW

File: docs/competition/volunteers/refreshments.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+People at these locations usually can't leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
 
 - Tech desks
 - Reception
 - Helpdesk
-- Arena: commentator, marshals, head shepherd, score entry
+- Arena, both on the ground and on the balcony.
 
-The tech desks roles and commentator are particularly important as they do not rotate between people at all.
+**Make sure you visit volunteers on the cube floor, in addition to those on the balcony, they were neglected last year!**
```
Comment (sedders123) [inline docs/competition/volunteers/refreshments.md:19]:
```suggestion
**Make sure you visit volunteers on the cube floor, in addition to those on the balcony, they were often forgotten last year!**
```
Makes it slightly less harsh?

**Review by sedders123 — APPROVED** [review]
Looks good to me! One potential tweak

File: docs/competition/volunteers/refreshments.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.
 
-People at the following locations are usually unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
+People at these locations usually can't leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
 
 - Tech desks
 - Reception
 - Helpdesk
-- Arena: commentator, marshals, head shepherd, score entry
+- Arena, both on the ground and on the balcony.
```
Comment (PeterJCLaw) [inline docs/competition/volunteers/refreshments.md:17]: nit:
```suggestion
- Arena, both on the ground and on the balcony
```

File: docs/competition/volunteers/refreshments.md [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+**In SR2023, the refreshments are in the Volunteer Rest room, which is on level 2 past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
 
-It is the responsibility of the refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.
+It is the responsibility of the refreshments role to offer refreshments to all volunteers, especially those who can't leave their roles.
 
 To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
 Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.
 
-People at the following locations are usually unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
+People at these locations usually can't leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
```
Comment (PeterJCLaw) [inline docs/competition/volunteers/refreshments.md:12]: Do we want to say something about there possibly being roles which may not be able to acknowledge the refreshments (e.g: livestream) and what to do (maybe just leave them some bottles of water if they don't have any, plus a selection of biscuits)?

**Pull request merged**